### PR TITLE
ML-377 ML-ConfTest doesn't compile

### PR DESCRIPTION
**Pull request by RogerDev**
Added #ifndef around structure in MakeR8Set
Also changed name of struct from work1 to pbb_cell to give it better uniqueness

Signed-off-by: Roger Dev <Roger.Dev@LexisNexis.com>

### CODE REVIEW

File: internal/MakeR8Set.ecl [hunk 1]
```diff
@@ -26,20 +26,23 @@ EXPORT SET OF REAL8 makeR8Set(dimension_t r, dimension_t s,
...
+
     #body
     __lenResult = r * s * sizeof(double);
     __isAllResult = false;
     double* result = (double*) rtlMalloc(__lenResult);
     //double * result = new double[r*s];
     __result = (void*) result;
-    work1 *cell = (work1*) d;
-    uint32_t cells = lenD / sizeof(work1);
+    work1 *cell = (pbb_cell*) d;
```
Comment (AttilaVamos) [inline internal/MakeR8Set.ecl]: I think it should be 
```
pbb_cell *cell = (pbb_cell*) d;
```

**Review by AttilaVamos — COMMENTED** [review]
I'm not sure this typedef stuff is compatible with our C++11 compiler environment. or something still missing.
I got this source and try to run it on one of our OBT but still get error message:
```
 'eclcc: /mnt/disk1/home/vamosax/.HPCCSystems/bundles/_versions/PBblas/V3_0_2/PBblas/internal/MakeR8Set.ecl(39,8): Error C6003: \xe2\x80\x98typedef\xe2\x80\x99 was ignored in this declaration [-Werror]
```

File: internal/MakeR8Set.ecl (hunk 1 above)
Comment (AttilaVamos) [reply internal/MakeR8Set.ecl]: And the right typedef is:
```
#ifndef pbb_cell
        typedef struct __attribute__ ((__packed__)) {      
          uint16_t wi_id;
          uint32_t x;
          uint32_t y;
          double v;
       } pbb_cell;
    #endif // pbb_cell
```
With this it is compile and run.

**Comment by RogerDev** [comment]
Fix per Attila.

**Comment by richardkchapman** [comment]
@AttilaVamos Please review

**Comment by AttilaVamos** [comment]
@RogerDev I can confirm this fix the problem. Good to merge.

**Comment by AttilaVamos** [comment]
@richardkchapman good to merge.

**Pull request merged**